admin/create-theme/theme-styles.php

```diff
@@ -46,7 +60,8 @@ public static function update_style_css( $style_css, $theme ) {
 
 		$css_metadata .= "Text Domain: {$text_domain}
 Tags: {$tags}
-*/
+
+{$copyright}*/
```

Comment:
How about wrapping this in an if statement? If no more than default metadata is present, a blank line is inserted before the comment's closing tag.

Below is an example for TT4.

![image](https://github.com/WordPress/create-block-theme/assets/54422211/34ab7675-f30f-4bc9-86b3-49d02123df7a)

Reply:
Refactored slightly to ensure the blank lines here are as expected.